diff --git a/BUILD.bazel b/BUILD.bazel
index 8644710645..8f4d7d1a7d 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -381,7 +381,7 @@ filegroup(
     srcs = [
         "src/Exception.i",
     ],
-    visibility = ["@//:__subpackages__"],
+    visibility = ["//:__subpackages__"],
 )
 
 filegroup(
@@ -389,7 +389,7 @@ filegroup(
     srcs = [
         "src/Exception-py.i",
     ],
-    visibility = ["@//:__subpackages__"],
+    visibility = ["//:__subpackages__"],
 )
 
 filegroup(
@@ -404,7 +404,7 @@ filegroup(
     srcs = [
         "src/options.i",
     ],
-    visibility = ["@//:__subpackages__"],
+    visibility = ["//:__subpackages__"],
 )
 
 sh_binary(
