diff -ru a/src/cts/BUILD b/src/cts/BUILD
--- a/src/cts/BUILD	2026-03-16 15:51:30.036108012 -0700
+++ b/src/cts/BUILD	2026-03-16 15:51:31.067333885 -0700
@@ -84,9 +84,11 @@
         "include/cts/MakeTritoncts.h",
     ],
     copts = [
-        "-Isrc/cts/src",
     ],
-    includes = ["include"],
+    includes = [
+        "src",
+        "include",
+    ],
     deps = [
         ":cts",
         ":private_hdrs",
diff -ru a/src/dbSta/BUILD b/src/dbSta/BUILD
--- a/src/dbSta/BUILD	2026-03-16 15:51:30.038452971 -0700
+++ b/src/dbSta/BUILD	2026-03-16 15:51:31.067643825 -0700
@@ -144,10 +144,10 @@
         "include/db_sta/MakeDbSta.hh",
     ],
     copts = [
-        "-Isrc/dbSta/src",
         "-Wno-missing-braces",  # from TCL swigging
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/dft/BUILD b/src/dft/BUILD
--- a/src/dft/BUILD	2026-03-16 15:51:30.040612311 -0700
+++ b/src/dft/BUILD	2026-03-16 15:51:31.068009165 -0700
@@ -48,9 +48,9 @@
         "include/dft/MakeDft.hh",
     ],
     copts = [
-        "-Isrc/dft/src",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/dpl/BUILD b/src/dpl/BUILD
--- a/src/dpl/BUILD	2026-03-16 15:51:30.042995950 -0700
+++ b/src/dpl/BUILD	2026-03-16 15:51:31.068357425 -0700
@@ -79,9 +79,9 @@
         "include/dpl/OptMirror.h",
     ],
     copts = [
-        "-Isrc/dpl/src",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/drt/BUILD b/src/drt/BUILD
--- a/src/drt/BUILD	2026-03-16 15:51:30.045096960 -0700
+++ b/src/drt/BUILD	2026-03-16 15:51:31.068773844 -0700
@@ -198,10 +198,10 @@
         "include/drt/TritonRoute.h",
     ],
     copts = [
-        "-Isrc/drt/src",
         "-fopenmp",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
@@ -261,10 +261,10 @@
         "include/drt/MakeTritonRoute.h",
     ],
     copts = [
-        "-Isrc/drt/src",
         "-Wno-missing-braces",  # from TCL swigging
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/est/BUILD b/src/est/BUILD
--- a/src/est/BUILD	2026-03-16 15:51:30.048071279 -0700
+++ b/src/est/BUILD	2026-03-16 15:51:31.069037144 -0700
@@ -71,11 +71,11 @@
         "include/est/MakeEstimateParasitics.h",
     ],
     copts = [
-        "-Isrc/est/src",
         "-Wno-missing-braces",  # from TCL swigging
         "-fopenmp",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/exa/BUILD b/src/exa/BUILD
--- a/src/exa/BUILD	2026-03-16 15:51:30.050862289 -0700
+++ b/src/exa/BUILD	2026-03-16 15:51:31.069260724 -0700
@@ -44,9 +44,9 @@
         "include/exa/MakeExample.h",
     ],
     copts = [
-        "-Isrc/exa/src",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/grt/BUILD b/src/grt/BUILD
--- a/src/grt/BUILD	2026-03-16 15:51:30.053809638 -0700
+++ b/src/grt/BUILD	2026-03-16 15:51:31.069513924 -0700
@@ -184,11 +184,11 @@
         "include/grt/MakeGlobalRouter.h",
     ],
     copts = [
-        "-Isrc/grt/src",
         "-Wno-missing-braces",  # from TCL swigging
         "-fopenmp",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/ifp/BUILD b/src/ifp/BUILD
--- a/src/ifp/BUILD	2026-03-16 15:51:30.056209387 -0700
+++ b/src/ifp/BUILD	2026-03-16 15:51:31.069790904 -0700
@@ -44,9 +44,9 @@
         "include/ifp/MakeInitFloorplan.hh",
     ],
     copts = [
-        "-Isrc/ifp/src",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/mpl/BUILD b/src/mpl/BUILD
--- a/src/mpl/BUILD	2026-03-16 15:51:30.058312977 -0700
+++ b/src/mpl/BUILD	2026-03-16 15:51:31.070040894 -0700
@@ -68,9 +68,9 @@
         "include/mpl/MakeMacroPlacer.h",
     ],
     copts = [
-        "-Isrc/mpl/src",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/odb/BUILD b/src/odb/BUILD
--- a/src/odb/BUILD	2026-03-16 15:51:30.060604327 -0700
+++ b/src/odb/BUILD	2026-03-16 15:52:10.334576439 -0700
@@ -137,12 +137,14 @@
     hdrs = ["include/odb/MakeOdb.h"],
     copts = [
         "-Wno-missing-braces",  # from TCL swigging
-        "-Isrc/odb/src/swig/common",
-        "-Isrc/odb/include",
     ],
     features = [
         "-use_header_modules",
     ],
+    includes = [
+        "include",
+        "src/swig/common",
+    ],
     deps = [
         ":odb",
         "//:ord",
diff -ru a/src/ppl/BUILD b/src/ppl/BUILD
--- a/src/ppl/BUILD	2026-03-16 15:51:30.063101836 -0700
+++ b/src/ppl/BUILD	2026-03-16 15:51:31.070729314 -0700
@@ -38,9 +38,9 @@
         "include/ppl/Parameters.h",
     ],
     copts = [
-        "-Isrc/ppl/src",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/rcx/BUILD b/src/rcx/BUILD
--- a/src/rcx/BUILD	2026-03-16 15:51:30.065837035 -0700
+++ b/src/rcx/BUILD	2026-03-16 15:51:31.070997934 -0700
@@ -106,9 +106,9 @@
         "include/rcx/MakeOpenRCX.h",
     ],
     copts = [
-        "-Isrc/rcx/src",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/rmp/BUILD b/src/rmp/BUILD
--- a/src/rmp/BUILD	2026-03-16 15:51:30.068227585 -0700
+++ b/src/rmp/BUILD	2026-03-16 15:51:31.071607214 -0700
@@ -35,10 +35,10 @@
         "include/rmp/Restructure.h",
     ],
     copts = [
-        "-Isrc/rmp/src",
     ],
     defines = ["ABC_NAMESPACE=abc"],
     includes = [
+        "src",
         "include",
     ],
     deps = [
@@ -76,9 +76,9 @@
         "include/rmp/MakeRestructure.h",
     ],
     copts = [
-        "-Isrc/rmp/src",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/rsz/BUILD b/src/rsz/BUILD
--- a/src/rsz/BUILD	2026-03-16 15:51:30.070518805 -0700
+++ b/src/rsz/BUILD	2026-03-16 15:51:31.072442474 -0700
@@ -93,9 +93,9 @@
         "include/rsz/MakeResizer.hh",
     ],
     copts = [
-        "-Isrc/rsz/src",
     ],
     includes = [
+        "src",
         "include",
     ],
     deps = [
diff -ru a/src/utl/BUILD b/src/utl/BUILD
--- a/src/utl/BUILD	2026-03-16 15:51:30.072783684 -0700
+++ b/src/utl/BUILD	2026-03-16 15:51:31.072874884 -0700
@@ -139,9 +139,11 @@
         "include/utl/MakeLogger.h",
     ],
     copts = [
-        "-Isrc/utl/src",
     ],
-    includes = ["include"],
+    includes = [
+        "src",
+        "include",
+    ],
     deps = [
         ":utl",
         "//:ord",
